src/hooks/useDSFRConfig.tsx

```diff
@@ -50,8 +50,13 @@ export const DSFRConfig = ({
       await import("@gouvfr/dsfr/dist/utility/utility.css");
       await import("@gouvfr/dsfr/dist/dsfr.css");
       // @ts-expect-error
-      window.dsfr.start();
```

Comment:
I think it's better to return early if typeof window === "undefined" in line 39 preferably.
Then here we could just call conditionally
`window.dsfr.start?.()`

Reply:
check now !